configure.ac

```diff
@@ -539,15 +541,26 @@ AS_IF([test ! "x${WISH}" = "x"],[
 
 ##### libpd #####
 AS_IF([test "x${enable_libpd}" = "xyes"],[
-        libpd="yes"
-        libpd_=""
-        AS_IF([test "x${enable_libpd_instance}" = "xyes"],[ libpd_="${libpd_}+multi "])
-        AS_IF([test "x${enable_libpd_utils}" = "xyes"],[ libpd_="${libpd_}+utils "])
-        AS_IF([test "x${enable_libpd_extra}" = "xyes"],[ libpd_="${libpd_}+extra "])
-        AS_IF([test "x${enable_libpd_setlocale}" = "xno"],[ libpd_="${libpd_}-setlocale "])
-        AS_IF([test "x${libpd_}" != "x"],[ libpd="${libpd} ( ${libpd_})"])
+    libpd="yes"
+    libpd_=""
+    AS_IF([test "x${enable_libpd_instance}" = "xyes"],[
+        libpd_="${libpd_}+multi "
+        AS_IF([test "x$MACOSX" = "xyes"],[
+            macos_version_min=10.9
+            AC_MSG_NOTICE([Setting macOS version min 10.9 for libpd instance])
+        ])
+    ])
+    AS_IF([test "x${enable_libpd_utils}" = "xyes"],[ libpd_="${libpd_}+utils "])
```

Comment:
oh, this breaks the nice layout and is much harder to parse.
i think it would be nicer to separate the `+multi` suffixing from the `macos_version_min` assignment:

~~~m4
AS_IF([test "x${enable_libpd_instance}" = "xyes"],[ libpd_="${libpd_}+multi "])
# ...
AS_IF([test "x${enable_libpd_instance}" = "xyes" && test "x$MACOSX" = "xyes"],[
    macos_version_min=10.9
    AC_MSG_NOTICE([Setting macOS version min 10.9 for libpd instance])
])
~~~

see also my other comment